configs/amcts/gatekeeper.cfg

```diff
@@ -0,0 +1,8 @@
+@include ../../engines/KataGo-raw/cpp/configs/training/gatekeeper1.cfg
```

Comment:
There's some risk this file changes underneath us and breaks things, but I think on balance it's probably better than duplicating it.